pkg/workload/workload.go

```diff
@@ -449,6 +449,8 @@ func StringTuple(datums []interface{}) []string {
 		switch x := datum.(type) {
 		case int:
 			s[i] = strconv.Itoa(x)
+		case uint64:
+			s[i] = fmt.Sprint(x)
```

Comment:
nit: I believe `strconv.FormatUint(x, 10)` is preferred.

Reply:
Any reason why that is preferred? Probably marginally faster, but I doubt that matters here.

Reply:
Yeah, it needs to do a lot less work and the `uint64` doesn't get passed in through a slice of empty interfaces. Probably not worth worrying about that here though. It does mirror the use of `strconv.Itoa` above.

Reply:
Done.